samples/openapi3/client/petstore/dart2/petstore_client_lib/lib/api/pet_api.dart

```diff
@@ -10,23 +10,24 @@
 
 part of openapi.api;
 
-
 class PetApi {
   PetApi([ApiClient? apiClient]) : apiClient = apiClient ?? defaultApiClient;
 
   final ApiClient apiClient;
 
   /// Add a new pet to the store
   ///
-  /// 
+  ///
   ///
   /// Note: This method returns the HTTP [Response].
   ///
   /// Parameters:
   ///
   /// * [Pet] pet (required):
   ///   Pet object that needs to be added to the store
-  Future<Response> addPetWithHttpInfo(Pet pet,) async {
+  Future<Response> addPetWithHttpInfo(
+    Pet pet,
```

Comment:
whitespace changes here too

Reply:
The newest version of this branch should have corrected the extraneous whitespace.